internal/services/securitycenter/security_center_server_vulnerability_assessment_resource.go

```diff
@@ -42,6 +41,8 @@ func resourceServerVulnerabilityAssessment() *pluginsdk.Resource {
 			Delete: pluginsdk.DefaultTimeout(10 * time.Minute),
 		},
 
+		DeprecationMessage: `This resource has been deprecated in favour of azurerm_security_center_vm_server_vulnerability_assessment and azurerm_security_center_hybrid_vm_server_vulnerability_assessment and will be removed in v3.0 of the AzureRM Provider`,
```

Comment:
At this point this is too late to be removed in 3.0, so this'll need to be removed in 4.0:

```suggestion
		DeprecationMessage: `This resource has been deprecated in favour of azurerm_security_center_vm_server_vulnerability_assessment and azurerm_security_center_server_vulnerability_assessment_arc_virtual_machine and will be removed in v4.0 of the AzureRM Provider`,
```